quick-start/go.mod

```diff
@@ -2,4 +2,17 @@ module github.gom/hashicorp/hello-vault-go/quick-start
 
 go 1.16
 
-require github.com/hashicorp/vault/api v1.7.1
+require (
+	github.com/cenkalti/backoff/v3 v3.2.2 // indirect
+	github.com/fatih/color v1.14.1 // indirect
+	github.com/google/go-cmp v0.5.9 // indirect
+	github.com/hashicorp/go-hclog v1.4.0 // indirect
+	github.com/hashicorp/go-retryablehttp v0.7.2 // indirect
+	github.com/hashicorp/go-secure-stdlib/parseutil v0.1.7 // indirect
+	github.com/hashicorp/vault/api v1.9.0
+	github.com/stretchr/testify v1.8.1 // indirect
+	golang.org/x/crypto v0.6.0 // indirect
+	golang.org/x/net v0.7.0 // indirect
+	golang.org/x/time v0.3.0 // indirect
+	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
+)
```

Comment:
what version of golang did you use? usually you see the direct and indirect deps separately

Reply:
I'm on 1.20.1 but this repo is on 1.16. Let me try bumping it up.